src/i18n/RuleEngine.js

```diff
@@ -1113,4 +1113,20 @@ export default {
     zh: '由于动作被禁用或在删除动作时出现竞态条件而被丢弃的动作数量。',
     en: 'The number of discarded actions due to action being disabled or race condition while action is being deleted.',
   },
+  buffer: {
+    zh: '缓冲',
+    en: 'Buffer',
+  },
+  queuedMessages: {
+    zh: '已缓存消息',
+    en: 'Queued Messages',
+  },
+  queueUsage: {
+    zh: '队列使用率',
```

Comment:
```suggestion
    zh: '队列使用量',
```

Is this not a “rate” here?